assets/atlas/SimpleFarming.atlas

```diff
@@ -12,7 +12,7 @@
             "", "", "", "", "", "", "", "",
             "", "", "", "", "", "", "", "",
             "", "", "", "", "", "", "", "",
-            "", "", "", "", "", "", "", ""
+            "", "", "", "", "", "", "BlackberrySeed", "Blackberry"
```

Comment:
You may want to just use standard images outside a texture atlas. While you're the only one I see who has actually modified the atlas (so no conflicts) it is a more fragile approach. Better suited when you have a big set of grouped images all by the same author for some unique work from said author, rather than a "communal" project so to say.

It does work though! So good work there. Up to you if you'd like to leave it this way or use standalone images.